// @mastra/memory/dist/observational-memory-4TV5KKFV.cjs
'use strict';

var chunkLGCREJMO_cjs = require('./chunk-LGCREJMO.cjs');



Object.defineProperty(exports, "OBSERVATIONAL_MEMORY_DEFAULTS", {
  enumerable: true,
  get: function () { return chunkLGCREJMO_cjs.OBSERVATIONAL_MEMORY_DEFAULTS; }
});
Object.defineProperty(exports, "OBSERVATION_CONTEXT_INSTRUCTIONS", {
  enumerable: true,
  get: function () { return chunkLGCREJMO_cjs.OBSERVATION_CONTEXT_INSTRUCTIONS; }
});
Object.defineProperty(exports, "OBSERVATION_CONTEXT_PROMPT", {
  enumerable: true,
  get: function () { return chunkLGCREJMO_cjs.OBSERVATION_CONTEXT_PROMPT; }
});
Object.defineProperty(exports, "OBSERVATION_CONTINUATION_HINT", {
  enumerable: true,
  get: function () { return chunkLGCREJMO_cjs.OBSERVATION_CONTINUATION_HINT; }
});
Object.defineProperty(exports, "OBSERVER_SYSTEM_PROMPT", {
  enumerable: true,
  get: function () { return chunkLGCREJMO_cjs.OBSERVER_SYSTEM_PROMPT; }
});
Object.defineProperty(exports, "ObservationalMemory", {
  enumerable: true,
  get: function () { return chunkLGCREJMO_cjs.ObservationalMemory; }
});
Object.defineProperty(exports, "TokenCounter", {
  enumerable: true,
  get: function () { return chunkLGCREJMO_cjs.TokenCounter; }
});
Object.defineProperty(exports, "buildObserverPrompt", {
  enumerable: true,
  get: function () { return chunkLGCREJMO_cjs.buildObserverPrompt; }
});
Object.defineProperty(exports, "buildObserverSystemPrompt", {
  enumerable: true,
  get: function () { return chunkLGCREJMO_cjs.buildObserverSystemPrompt; }
});
Object.defineProperty(exports, "extractCurrentTask", {
  enumerable: true,
  get: function () { return chunkLGCREJMO_cjs.extractCurrentTask; }
});
Object.defineProperty(exports, "formatMessagesForObserver", {
  enumerable: true,
  get: function () { return chunkLGCREJMO_cjs.formatMessagesForObserver; }
});
Object.defineProperty(exports, "getObservationsAsOf", {
  enumerable: true,
  get: function () { return chunkLGCREJMO_cjs.getObservationsAsOf; }
});
Object.defineProperty(exports, "hasCurrentTaskSection", {
  enumerable: true,
  get: function () { return chunkLGCREJMO_cjs.hasCurrentTaskSection; }
});
Object.defineProperty(exports, "optimizeObservationsForContext", {
  enumerable: true,
  get: function () { return chunkLGCREJMO_cjs.optimizeObservationsForContext; }
});
Object.defineProperty(exports, "parseObserverOutput", {
  enumerable: true,
  get: function () { return chunkLGCREJMO_cjs.parseObserverOutput; }
});
